python/ribasim/ribasim/validation.py

```diff
@@ -0,0 +1,46 @@
+# Table for connectivity
```

Comment:
Are you planning on generating this dict automatically?

Reply:
Since the current change are a bit large, is it okay to make it into an new issue?

Reply:
I think it would be good to do it in this PR, as it would otherwise just be the next issue to pick up? I can review the rest in between.

Reply:
Auto-generating is a different scope from validation. I still think it would be better to do it in a different issue.